examples/seismic/model.py

```diff
@@ -389,14 +396,21 @@ def initialize_damp(damp, nbpml, spacing, mask=False):
 
 
 def initialize_function(function, data, nbpml, pad_mode='edge'):
-    """Initialize a :class:`Function` with the given ``data``. ``data``
+    """
+    Initialize a `Function` with the given ``data``. ``data``
     does *not* include the PML layers for the absorbing boundary conditions;
     these are added via padding by this function.
 
-    :param function: The :class:`Function` to be initialised with some data.
-    :param data: The data array used for initialisation.
-    :param nbpml: Number of PML layers for boundary damping.
-    :param pad_mode: A string or a suitable padding function as explained in
+    Parameters
+    ----------
+    function : Function
+        to be initialised with some data.
+    data : ndarray
+        The data array used for initialisation.
+    nbpml : int
+        Number of PML layers for boundary damping.
+    pad_mode : str or padding function FIX
```

Comment:
I still see the word "FIX" here

Reply:
It is meant to be a reminder to check with you whether it is right. Is it OK like this?

Reply:
I'd write:
```
pad_mode : str or callable, optional
```

Reply:
Fixed